src/components/victory-axis.jsx

```diff
@@ -47,9 +47,17 @@ const styles = {
   }
 };
 
-class VAxis extends React.Component {
+@Radium
+export default class VictoryAxis extends React.Component {
+  static role = "axis";
```

Comment:
hmm, as far as I can tell `role: "axis"` /  `VictoryAxis.prototype.role = "axis"` does not us access to the role via `child.role`. Inspecting the children, and what I'm able to access from them, the look ups are :

```
static role = "axis"  // chlid.type.role = "axis"
role: "axis" // child.type.prototype.role = "axis"
```
Neither case exposes `role` on the top level of the child object. I think maybe if we were accessing children via `ref` we could get that behavior, but I'm really not sure. The docs around this are pretty thin, which makes me think this is not a blessed pattern.